### DISCUSSION TITLE
Unable to unmount drive

### DISCUSSION THREAD

**Discussion by EarsofTin** [rern/rAudio#293] [Bug Reports]
Running 20210403 update on a fresh image.  Unmount button results in no action.  Combined with the "add to playlist" problem it seems there is a decoupling of android browser and the the player.  As soon as I have time I will add a monitor/mouse to the player and see if problems persist.

**Comment by apastuszak** [comment]
I'm having the same issue.  Need to unmount a share, as I migrated my music to a NAS.  Is there some way to remove this from the bash prompt.

**Reply by madmic1314** [reply]
Same here in the latest build 20210703. Is there a command way of doing this? Will have a look at that later on.
